.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Function Release
+on:
+  push:
+    tags:
+    # e.g. functions/go/apply-setters/v1.1.1
+    - "functions/*/*/v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
What about the contrib functions?

Reply:
Do they use a different tag format?

Reply:
We haven't released many contrib functions yet. `sops` was the only one that has been released. But I imagine that tags like `contrib/functions/go/foo/v0.1.0` (matching its path) will be used.

Reply:
Okay, I'll add support for the contrib path as well in that case. For contrib functions do we want the short tag to be the same `<name>/<version>` format?

Reply:
Yes